Game/Assets/Scripts/src/Script/CharacterSelectionMenu.h

```diff
@@ -0,0 +1,52 @@
+#ifndef  __CHARATERSELECTIONMENU_H
+#define  __CHARATERSELECTIONMENU_H
+
+#include "Script.h"
+class ComponentAudioSource;
+class CharacterSelectionMenu : public Script
+{
+
+public:
+	CharacterSelectionMenu();
+	~CharacterSelectionMenu() = default;
+
+	void Awake() override;
+	void Update() override;
+
+	void SelectCharacter();
+
+	void OnInspector(ImGuiContext * context) override;
+
+	void InitPublicGameObjects() override;
+private:
+	std::vector<GameObject*> buttons;
+	unsigned current = 0;
+	bool player1_choice = true; //TODO: CHANGE THIS FOR THE MODULE INPUT VARIABLE
```

Comment:
What you have to change is the static bool of worldmanager player1_choice.

Reply:
yes! I mess it up with the module input